CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Change Log
 
+## [0.0.34] - 2024-05-16
```

Comment:
Hi Vidya Thanks for this PR, usually changelog are separate PR's from the actual PR's. was that an intention or this file commit was by mistake?

Reply:
oh yeah it was by mistake! I didn't realize I added this, I was more focused on other files and didn't pay attention to changelog.